pkg/sql/logictest/testdata/logic_test/inverted_index

```diff
@@ -1689,5 +1707,5 @@ SELECT primes FROM cb WHERE primes && numbers ORDER BY primes
 
 # Regression test for incorrectly unwrapping a DOidWrapper (#84569).
 statement ok
-CREATE TABLE t84569 (name_col NAME NOT NULL, INVERTED INDEX (name_col gin_trgm_ops DESC));
```

Comment:
I've confirmed that the bug was reproducible without `DESC`, so this regression test remains useful.